src/filters/index.js

```diff
@@ -63,20 +63,23 @@ export default {
   /**
    * 12345 => $12,345.00
    *
-   * @param {String} sign
+   * @param {String} currency
+   * @param {Number} places
    */
 
-  currency (value, currency) {
+  currency (value, currency, places) {
```

Comment:
I'd rather call places decimals, but I'm not native English so I prefer someone to confirm or reject this